homeassistant/components/wink.py

```diff
@@ -194,6 +194,8 @@ def device_state_attributes(self):
             attributes["device_manufacturer"] = self._device_manufacturer
         if self._model_name:
             attributes["model_name"] = self._model_name
+        if self._tamper is not None:
```

Comment:
The code is fine as is and I'm down with merging this. Just one final improvement that you could do.

By having `self._tamper` be a property that you call twice, you're also running that code twice. To avoid running it twice, you can either inline the property here (as it's not used by anything else anyway) or you can cache the call to the property. `tamper = self._tamper; if tamper is not None:`